esp/templates/survey/questions/long_answer.html

```diff
@@ -2,7 +2,9 @@
 {% load survey %}
 <table class="fullwidth">
 <tr>
-    <td width="35%"><b>{{ name }}</b></td>
-    <td><textarea rows="{{ rows }}" name="question_{{ id }}{% if for_class %}_{% templatetag openvariable %} id {% templatetag closevariable %}{% endif %}"></textarea></td>
+    <td>
+    <div style = "text-align: left; font-weight: bold;"> {{ name }} </div>
+    <div style="text-align: left; padding-left:2em; text-indent:-2em;"><textarea rows="{{ rows }}" name="question_{{ id }}{% if for_class %}_{% templatetag openvariable %} id {% templatetag closevariable %}{% endif %}"></textarea></div>
```

Comment:
It might make sense to give this `width: 100%`, since there's no way to specify the default width of the box.